template/deploy/helm/[[product]]-operator/templates/roles.yaml.j2

```diff
@@ -73,6 +73,18 @@ rules:
     resources:
       - pods
       - configmaps
+      - services
+      - endpoints
+    verbs:
```

Comment:
The verb `patch` must be added to `services`, `statefulsets`, and `jobs`.
The verbs should be ordered alphabetically.

Reply:
9c0c974ba6b9192d05001d4a2b3df7c5cbc6ce40